src/runner.ts

```diff
@@ -0,0 +1,86 @@
+import * as core from "@actions/core";
+
+export type CoberturaCoverageNode = {
+    lineRate: number;
+    linesCovered: number;
+    linesValid: number;
+    packages: Map<string, CoberturaPackageNode>;
```

Comment:
I see that we are using a map to store multiple same type of elements as a design. My question is, why not use an array of objects instead? Do we have any scenarios where we need to use the name to find the corresponding object?

Reply:
This is used by the Azure plug-in: https://github.com/parasoft/parasoft-findings-vsts/blob/53f7f6f079a7ca813d39f0b20eb2c316dfbc70dd/PublishParasoftResults/CoverageReportService.ts#L31

Reply:
> I see that we are using a map to store multiple same type of elements as a design. My question is, why not use an array of objects instead? Do we have any scenarios where we need to use the name to find the corresponding object?

Currently, we are keeping the Map for future use if we support multiple coverage reports. If only one report is needed, we can change it to use an array of objects.